vignettes/_translation_links.R

```diff
@@ -0,0 +1,25 @@
+# build a link list of alternative languages (may be character(0))
+.write.translation.links <- function(fmt) {
+    url = "https://rdatatable.gitlab.io/data.table/articles"
+    path = dirname(knitr::current_input(TRUE))
```

Comment:
is there a litedown equivalent (#6583)?

Reply:
The only alternative I found was `litedown:::.env$input`, that stores the file name (the input path to `litedown::fuse()`). Tested to work building the package from . and from ..